tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -35,7 +35,7 @@ var _ = Describe("odo devfile push command tests", func() {
 	When("creating a nodejs component", func() {
 		output := ""
 		BeforeEach(func() {
-			helper.Cmd("odo", "create", "--project", commonVar.Project, cmpName, "--devfile", helper.GetExamplePath("source", "devfiles", "nodejs", "devfile-registry.yaml")).ShouldPass()
+			helper.Cmd("odo", "init", "--name", cmpName, "--devfile-path", helper.GetExamplePath("source", "devfiles", "nodejs", "devfile-registry.yaml")).ShouldPass()
```

Comment:
Feel free to completely remove all the problematic tests from this file (or all the file), as this file will be removed in the other PR #5570